browser/about_flags.cc

```diff
@@ -311,6 +315,13 @@ constexpr char kRestrictWebSocketsPoolDescription[] =
 #define BRAVE_VPN_FEATURE_ENTRIES
 #endif
 
+#define BRAVE_SKU_SDK_FEATURE_ENTRIES                  \
```

Comment:
so actually do we want to expose this in `about://flags`? I don't think we want any users turning this on yet, right?

Reply:
Since it's limited to Brave pages (dev ones behind auth), I think it's OK to enable. Won't really cause a problem. And now that we're tying VPN to it, we'll need a way to turn VPN on (which means turning SKU on also)

Reply:
They can also be enabled by command line. @bbondy wdyt about this? Should we be exposing this kind of stuff in `brave://flags/`?